### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by wa12rior**
I ve updated documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Project name
+## Requirements
+## Installation
```
Comment (wa12rior) [inline README.md:3]: Remove please.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Project name
```
Comment (b4kii) [inline README.md:1]: ala ma kota

File: README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+## Requirements
```
Comment (b4kii) [inline README.md:2]: ala ma kota

File: README.md (hunk 2 above)
Comment (wa12rior) [reply README.md:1]: nie, zmień to please

**Pull request merged**